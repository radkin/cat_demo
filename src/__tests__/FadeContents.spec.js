import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import FadeContents, { Counter, dataReducer } from '../DropdownContainer/FadeContents';

describe('FadeContents', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<FadeContents />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
