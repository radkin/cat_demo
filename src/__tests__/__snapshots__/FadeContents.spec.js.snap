// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FadeContents snapshot renders 1`] = `
<div
  className="sc-AxjAm bDJkhE"
/>
`;
